rag_experiment_accelerator/doc_loader/pdfLoader.py

```diff
@@ -49,7 +49,7 @@ def _load_pdf_with_pypdf(file_path: str) -> List[Document]:
 
 def _load_pdf_with_unstructured(file_path: str) -> List[Document]:
     logger.debug("Loading PDF file with unstructured: %s", file_path)
-    elements = partition_pdf(filename=file_path, strategy="ocr_only")
+    elements = partition_pdf(filename=file_path, strategy="hi_res")
     element_meta = elements[0].to_dict()["metadata"] if elements else {}
```

Comment:
**[Severity: Major]** The change from `strategy="ocr_only"` to `strategy="hi_res"` may impact the performance and accuracy of PDF content extraction. Ensure that this change aligns with the intended functionality and document the rationale behind this choice for future maintainers.


---

**[peer-review-pal]** We are sorry for the intrusion. Feel free to ignore or delete the comments/issue with the tag - they are automatically generated by our [Hack project](https://hackbox.microsoft.com/hackathons/hackathon2024/project/81776) - repository **[Peer Review Pal](https://github.com/microsoft/peer-review-pal)**.
If you find them useful, want us to delete them, have any feedback at all or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding **#prp-ignore** to your PR description.